gravitee-apim-console-webui/src/components/gio-side-nav/gio-side-nav.component.spec.ts

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { HttpTestingController } from '@angular/common/http/testing';
+import { HarnessLoader } from '@angular/cdk/testing';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { MatIconTestingModule } from '@angular/material/icon/testing';
+import { InteractivityChecker } from '@angular/cdk/a11y';
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { GioLicenseExpirationNotificationHarness, LICENSE_CONFIGURATION_TESTING } from '@gravitee/ui-particles-angular';
+import { ActivatedRoute } from '@angular/router';
+import { of } from 'rxjs';
+
+import { GioSideNavHarness } from './gio-side-nav.harness';
+import { GioSideNavComponent } from './gio-side-nav.component';
+import { GioSideNavModule } from './gio-side-nav.module';
+
+import { License } from '../../entities/license/License';
+import { GioPermissionService } from '../../shared/components/gio-permission/gio-permission.service';
+import { CONSTANTS_TESTING, GioHttpTestingModule } from '../../shared/testing';
+describe('GioSideNavComponent', () => {
+  let fixture: ComponentFixture<GioSideNavComponent>;
+  let httpTestingController: HttpTestingController;
+  let loader: HarnessLoader;
+
+  const expirationDateInOneYear = new Date();
+  expirationDateInOneYear.setFullYear(expirationDateInOneYear.getFullYear() + 1);
+
+  const init = async (licenseNotificationEnabled = true, hasLicenseMgmtPermission = true) => {
+    await TestBed.configureTestingModule({
+      declarations: [GioSideNavComponent],
+      imports: [NoopAnimationsModule, GioHttpTestingModule, GioSideNavModule, MatIconTestingModule],
+      providers: [
+        {
+          provide: GioPermissionService,
+          useValue: {
+            hasAnyMatching: (permissions: string[]) => {
+              return !(!permissions.includes('organization-license_management-r') || !hasLicenseMgmtPermission);
```

Comment:
Can this be simplified by removing negation ?

Reply:
My IDE simplified my if/then to this, but I agree it's a bit negative-y and compact 🙈

Reply:
Done